Scripts/Python/plasma/PlasmaGame.py

```diff
@@ -41,79 +41,150 @@
 
  *==LICENSE==* """
 
+# NOTE: This stub file was generated automatically from Plasma's Python interface.
+# Do not edit this file manually.
+# To change any of the docstrings or function signatures,
+# edit the corresponding C++ glue code in pfPython.
+# If the Python interface has changed, regenerate these stubs
+# by running the following call in the in-game Python console:
+# >>> __import__("generate_stubs").run()
+
 from __future__ import annotations
 from typing import *
 
 class ptGameCli:
     """Abstract bass class for legacy game clients."""
 
-    gameID: int = ...
-    """The ID of the game instance on the server."""
+    gameID: Any
+    """Gets the ID of the game instance on the server."""
```

Comment:
For things that are implemented as properties, it would be nice if the comments didn't start with "Gets" (but I assume that's something to fix on the C++ docstring side?)

Reply:
Yes, this comes from the docstrings defined in C++. I'll adjust that soon (along with some other issues in the C++ docstrings).